tests/static/all_question_types.xml

```diff
@@ -85,6 +85,7 @@
 					</text>
 					<text id="question1-help">
 						<value>question1 en help</value>
+						<value form="markdown">question1 en help</value>
```

Comment:
Would it make sense to omit the markdown itext form if it's identical to the non-markdown version? Would that add a lot of complexity? Seems like wasted bandwidth. Might want to check with @ctsims on this.